README.md

```diff
@@ -74,14 +74,18 @@ script/test
 Classroom for GitHub uses environmental variables for storing credentials used in testing, these values are located in your `.env` file (created by `script/setup`).
 If you are recording new cassettes, you need to make sure all of these values are present.
 
+For the test environment you will need 2 user accounts and an organization (preferably one created specifically for testing against). The first user is the admin of the organization, the other is the student for the course.
+
 ENV Variable | Description |
 :-------------------|:-----------------|
-`TEST_CLASSROOM_OWNER_ID` | The GitHub `user_id` of an organization admin.
-`TEST_CLASSROOM_OWNER_GITHUB_TOKEN` | The [Personal Access Token](https://github.com/blog/1509-personal-api-tokens) for the classroom owner
-`TEST_CLASSROOM_STUDENT_ID` | Test OAuth application client ID.
-`TEST_CLASSROOM_STUDENT_GITHUB_TOKEN` | The [Personal Access Token](https://github.com/blog/1509-personal-api-tokens) for the student
-`TEST_CLASSROOM_OWNER_ORGANIZATION_ID` | GitHub ID (preferably one created specifically for testing against).
-`TEST_CLASSROOM_OWNER_ORGANIZATION_LOGIN` | GitHub login (preferably one created specifically for testing against).
+`TEST_CLASSROOM_OWNER_ID` | The GitHub `user_id` (number) of an organization admin.
```

Comment:
Going back and looking at this again, I'm not sure I agree with adding `(number)` to this description, maybe it's just me (and it very well could be) but I don't think `(number)` is needed.